### DISCUSSION TITLE
seek help! How to find the list of files imported by webpack?

### DISCUSSION THREAD

**Discussion by wangjieCode** [webpack/webpack#15889] [Q&A] [answered]
I want to find files that are not referenced in the project.How can I find the list of files imported by webpack in webpack to help me complete this requirement?

**Comment by vankop** [comment, marked as answer]
there is list of modules in stats https://webpack.js.org/configuration/stats/#statsmodules

**Reply by wangjieCode** [reply]
thank you

**Reply by wangjieCode** [reply]
I want to know why there are files used for webpack statistics?

**Reply by vankop** [reply]
what you mean? you can use webpack api to get stats object
